content/terraform/v1.14.x (alpha)/docs/cli/commands/query.mdx

```diff
@@ -0,0 +1,86 @@
+---
+page_title: locals block reference for `.tfquery.hcl` files
+description: |-
+ Add the `locals` block to your `.tfquery.hcl` files define values to reuse within the local Terraform module. 
+---
+
+# `terraform query` command
+
+The `terraform query` command queries existing infrastructure for unmanaged resources according to the `tfquery.hcl` file so that you can import them into your Terraform workspace.
+
+## Usage
+
+```shell-session
+$ terraform query [flags] [options]
+```
+
+## Description
+
+When you run the `terraform query` command, Terraform searches the current configuration directory for `tfquery.hcl` files so that it can query remote infrastructure for resources that match the defined `list` blocks. Terraform prints the results to the terminal. You also add the `-generate-config-out` flag to generate configuration that you can use to import the resources. Refer to [Import resources in bulk](/terraform/language/import/bulk) for more information.
```

Comment:
```suggestion
When you run the `terraform query` command, Terraform searches the current configuration directory for files ending in `.tfquery.hcl` so that it can query remote infrastructure for resources that match the defined `list` blocks. Terraform prints the results to the terminal. You can also add the `-generate-config-out` flag to generate configuration that you can use to import the resources. Refer to [Import resources in bulk](/terraform/language/import/bulk) for more information.
```

Reply:
At least in my testing, Terraform picked up the file name `main.tfquery.hcl`. If this has changed from the branch I'm working off of please ignore.